core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncManager.java

```diff
@@ -113,11 +113,12 @@ public boolean isActivelySynced(AlluxioURI path) throws InvalidPathException {
    * Adds a syncpoint from a journal entry.
    *
    * @param syncPoint sync point
+   * @param mountId mount id
    * @return true if sync point successfully added
    */
-  public boolean addSyncPointFromJournal(AlluxioURI syncPoint)
+  public boolean addSyncPointFromJournal(AlluxioURI syncPoint, long mountId)
       throws InvalidPathException, IOException {
-    return startSyncInternal(syncPoint);
+    return startSyncFromJournal(syncPoint, mountId);
```

Comment:
We shouldn't be starting anything from the journal. Can we keep journal replay as simple as possible and move thread startup to the `master.start()` method?

Reply:
the name may be misleading, but `startSyncFromJournal` actually only changes a few states, it does not startup threads.